<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineControlWindow</class>
 <widget class="QWidget" name="EngineControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EngineControlWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{

	font: 14pt &quot;Sans Serif&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_20">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
border: 2px solid line black;
	
	background-color: rgb(126, 125, 112);
}

QSlider
{
background-color: rgb(85, 85, 127);
border: 2px solid line black;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_54">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	background-color: rgb(67, 136, 101);

}</string>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="ButMovEng1Up">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/MainControllerClass/TrainerPictures/право1.png</normaloff>:/MainControllerClass/TrainerPictures/право1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineXPos1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSlider" name="SliderEngine1X">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-120000</number>
        </property>
        <property name="maximum">
         <number>120000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QSlider" name="SliderEngine1Y">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-120000</number>
        </property>
        <property name="maximum">
         <number>120000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineYPos1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="ButMovEng1Right">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/MainControllerClass/TrainerPictures/право1.png</normaloff>:/MainControllerClass/TrainerPictures/право1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="ButEngineToNull1">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(200, 200, 200);
}</string>
        </property>
        <property name="text">
         <string>В нуль</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_25">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	background-color: rgb(170, 170, 0);

}</string>
        </property>
        <property name="text">
         <string>с прибора</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="LabelCurrentYPos1">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	background-color: rgb(186, 186, 186);

}</string>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="LabelCurrentXPos1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	background-color: rgb(186, 186, 186);

}</string>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_24">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	background-color: rgb(170, 170, 0);

}</string>
        </property>
        <property name="text">
         <string>секунды</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_55">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	background-color: rgb(67, 136, 101);

}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="ButMovEng1Left">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/MainControllerClass/TrainerPictures/лево1.png</normaloff>:/MainControllerClass/TrainerPictures/лево1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="ButMovEng1Down">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/MainControllerClass/TrainerPictures/лево1.png</normaloff>:/MainControllerClass/TrainerPictures/лево1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
border: 2px solid line black;
	
	background-color: rgb(255, 170, 0);
}</string>
        </property>
        <property name="text">
         <string>Привод</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="checkWorkBlock">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox
{
border: 2px solid line black;
	
	
	background-color: rgb(208, 0, 0);
}

QCheckBox::checked
{
border: 2px solid line black;
	
	background-color: rgb(85, 170, 0);
}</string>
        </property>
        <property name="text">
         <string>Работа</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="LabelNumberChannel">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="butTurnOffHorizontal">
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	
	background-color: rgb(161, 36, 25);
	color: rgb(0, 0, 0);
}

QPushButton:checked
{
		background-color: rgb(61, 121, 0);
	
}</string>
          </property>
          <property name="text">
           <string>Горизонталь</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butTurnOffVertical">
          <property name="styleSheet">
           <string notr="true">QPushButton
{
		background-color: rgb(126, 0, 0);

}

QPushButton:checked
{
	
	background-color: rgb(61, 121, 0);
}</string>
          </property>
          <property name="text">
           <string>Вертикаль</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditCommandDelay">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Задержка</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>мкс</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
